forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -1802,12 +1804,52 @@ public void addCounterInternal(final CounterType counterType, final int n, final
         }
     }
 
-    public boolean addCounterTimestamp(CounterType counterType) {
-        return addCounterTimestamp(counterType, true);
+    public Card getCounterEffect(CounterType counterType, long timestamp) {
+        Card result = counterTypeKeywordEffects.get(counterType);
+
+        int num = 1;
+        if (!Keyword.smartValueOf(counterType.toString().split(":")[0]).isMultipleRedundant()) {
+            num = getCounters(counterType);
+        }
+
+        String keywords = Collections.nCopies(num, counterType.toString()).stream().collect(Collectors.joining(" & "));
+
+        if (result == null) {
+            result = SpellAbilityEffect.createEffect(castSA, this, this.getController(), counterType.toString(), this.getImageKey(), timestamp);
+            result.setRenderForUI(false);
+            result.addRemembered(this);
+
+            String s = "Mode$ Continuous | AffectedDefined$ Remembered | AddKeyword$ " + keywords;
+            StaticAbility stAb = result.addStaticAbility(s);
+            stAb.setActiveZone(EnumSet.of(ZoneType.Command));
+
+            // TODO add for zone changes, not just Battlefield?
+            if (isInPlay()) {
+                GameCommand until = SpellAbilityEffect.exileEffectCommand(getGame(), result);
+                addLeavesPlayCommand(until);
+            } else {
+                SpellAbilityEffect.addForgetOnMovedTrigger(this, getZone().getZoneType().toString());
+            }
+
+            getGame().getAction().moveToCommand(result, null);
+
+            counterTypeKeywordEffects.put(counterType, result);
+        }
+
+        // update timestamp
+        result.setGameTimestamp(timestamp);
+
+        // update amount
+        for (StaticAbility stAb : result.getStaticAbilities()) {
```

Comment:
this can be else?

Reply:
My goal was to add something to StaticAbilityContinous that would Multiply the AddKeywords,
so i just would need to update the SVar on the Effect.

But i didn't found the right way yet